code-react/src/views/dashboard/app/user-profile-edit.js

```diff
@@ -1,11 +1,81 @@
 import React from 'react'
 import {Container, Row, Col, Card, Tab, Form, Button, Nav} from 'react-bootstrap'
-import { Link } from 'react-router-dom'
+// import { Link } from 'react-router-dom'
+import { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { useSelector } from 'react-redux';
+import { useGetUserInfoQuery, useUpdateUserMutation } from '../../../store/slices/profileApiSlice';
+
+
+
+const familyMembers = {
+    id: "65a9328c79bd3261ba6ac4ac",
+    spaceName: "Gashlan",
+    Members: [
+        {
+            id: "65a9328c79bd3261ba6ac4aa",
+            firstName: "Ayman",
+            lastName: "Gashlan",
+            userName: "gashlan",
+            email: "agashlan1991@gmail.com",
+            gender: "male",
+            dateOfBirth: "1991-05-22T00:00:00.000Z",
+            active: true,
+            activationCode: "15550",
+            role: "parent",
+            family: "65a9328c79bd3261ba6ac4ac",
+            createdAt: "2024-01-18T14:15:40.345Z",
+            updatedAt: "2024-02-01T06:03:42.512Z"
+        },
+        {
+            id: "65bb5c5bcb6c4397255aef3a",
+            firstName: "XYZ",
+            lastName: "ASD",
+            userName: "admin2",
+            email: "asd@asd.com",
+            gender: "female",
+            dateOfBirth: "2024-05-05T00:00:00.000Z",
+            active: false,
+            activationCode: "62896",
+            role: "parent",
+            family: "65a9328c79bd3261ba6ac4ac",
+            createdAt: "2024-02-01T08:54:51.435Z",
+            updatedAt: "2024-02-01T08:54:51.435Z"
+        }
+    ],
+    createdAt: "2024-01-18T14:15:40.448Z",
+    updatedAt: "2024-02-01T08:54:51.528Z"
+}
+
+
+
 
-//image
-import img1 from '../../../assets/images/user/11.png'
 
 const UserProfileEdit =() =>{
+    
+    const { data: userInfo, refetch } = useGetUserInfoQuery();

```

Comment:
I just noticed that you are using a query to fetch userInfo 

Its betttwer to use the userInfo that is inside the state
const { userInfo } = useSelector(state => state.authReducer) || {};

this will require less fetches 


please note that becuase now backend and frontend are all in your same machine, it seems like its everything is fast, but when they are but into servers , its recommended to minimize fetches to the backend